src/views/alt_title.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * © 2019 Atypon Systems LLC
```

Comment:
`2025` here and in the other newly created file.

Reply:
Fixed. Thanks